vue/components/ui/molecules/upload-button/upload-button.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <div
+        class="upload-button"

```

Comment:
Not meant to make you miserable but I believe this is usually the other way around: `button-upload` (e.g.: `input-image`).